openvpn/check.go

```diff
@@ -10,7 +10,7 @@ import (
 // CheckOpenvpnBinary function checks that openvpn is available, given path to openvpn binary
 func CheckOpenvpnBinary(openvpnBinary string) error {
 
-	process := NewProcess(openvpnBinary, "[openvpn binary check]")
+	process := NewProcess(openvpnBinary, "[openvpn binary check] ")
```

Comment:
This is out of scope of this PR, but I hate that we're doing this prefix formatting of `"[%s] "` in so many places - it should be done in one place :/

Reply:
Well [] it's just nice surround. I don't see any problem not to use [ ] at all.

Reply:
Using it is good, it increases readability - but I think this formatting should be done in one place so that we could easily change that. Not this formatting is duplicated across whole system.